### DISCUSSION TITLE
Question about setting up a project using git.

### DISCUSSION THREAD

**Discussion by budhines** [community/community#23314] [New to GitHub]
<p>I have a folder for all of my code base in ColdFusion.</p>
<p>The folder looks like this: c:/ColdFusion2016/cfusion/wwwroot/ </p>
<p>Under the wwwroot I have a number of projects which are completely separate applications:</p>
<p>Belairautoservice</p>
<p>DonnasMarket</p>
<p>JqueryTestSamples</p>
<p>learncf_in_a_week</p>
<p>savemybookmarks</p>
<p>etc.</p>
<p>Do I navigate to the above cfusion/wwwroot and run git init </p>
<p>OR</p>
<p>Do I navigate to each of the project folders like Belairautoservice and then run git init?</p>
<p>Thank you</p>
<p>Question: </p>

**Comment by airtower-luna** [comment, marked as answer]
<p>It’s best practice to keep each project in a separate repository, so you should do “git init” inside the project folders.</p>

**Comment by budhines** [comment]
<p>Thank you for the quick response and the explanation.</p>
